src/components/product-property/index.js

```diff
@@ -0,0 +1,54 @@
+/* eslint-disable react/no-array-index-key */
+// @flow
+
+import React from 'react';
+import Icon from '../icon';
+import { Text } from '../typography';
+import { Container, ListContainer } from './index.style';
+
+type Props = {
+  iconName: string,
+  text: string,
+}
+
+type ComponentPros = {
+  list: [],
+  isHorizontal?: boolean,
+}
+
+const ProductProperty = (props: Props) => {
```

Comment:
ProductProperty component doesn't export. So we can't use this component in other files if we need it.

Reply:
we don't need this component anymore, if one item want to appear product-properties render a single product-property component, without ul li